src/profile-logic/js-tracer.js

```diff
@@ -528,6 +529,10 @@ export function convertJsTracerToThreadWithoutSamples(
   // Build up maps between index values.
   const funcMap: Map<IndexIntoStringTable, IndexIntoFuncTable> = new Map();
   const stackMap: Map<IndexIntoJsTracerEvents, IndexIntoStackTable> = new Map();
+  const frameMap: Map<
+    IndexIntoJsTracerEvents,
+    IndexIntoStringTable | null
+  > = new Map();
```

Comment:
Hm, at first look, I couldn't understand the use of this map from its name. Thought it's similar to `funcMap` and `stackMap` but it's not quite the case. Not sure what would be better though, maybe `frameImplementationMap` or something? What do you think?

Reply:
Thanks! I think `implementationMap` should be accurate enough. I also added a comment to help clarify the situation.

```
  // The implementationMap maps to index in the string table, that refers to a frame's
  // implementation. e.g. the number 132 which maps to the string "IonMonkey".
  const implementationMap: Map<
    IndexIntoJsTracerEvents,
    IndexIntoStringTable | null
  > = new Map();
```